configuration/concepts/adult_hiv_followup.csv

```diff
@@ -0,0 +1,72 @@
+462e1a64-6d23-4590-9393-c5fdcba78a42,,,Retinite a CMV (Date),CMV Retinitis (Date),Retinite a CMV (Date),CMV Retinitis (Date),Misc,Date,,
+ecaf7d8e-161d-4e28-9ce8-b3b7dd9772bb,,,Sarcome de Kaposi (Date),Kaposi's sarcoma (Date),Sarcome de Kaposi (Date),Kaposi's sarcoma (Date),Misc,Date,,
+c6658a5c-91c9-4dd9-b9a2-d002a139dc97,,,Herpés (Date),Herpes (Date),Herpés (Date),Herpes (Date),Misc,Date,,
+20f7c749-9827-4508-a820-c134ebb3e271,,,Candidose oesophagienne (Date),Esophageal candidiasis (Date),Candidose oesophagienne (Date),Esophageal candidiasis (Date),Misc,Date,,
+175ae517-88f7-4c77-a07d-5bc149fcab3c,,,Isosporose (Date),Isosporosis (Date) ,Isosporose (Date),Isosporosis (Date) ,Misc,Date,,
+673ce6c2-9d44-48b2-8888-ef656cf1ccfd,,,Toxoplasmose cérébrale (Date),Cerebral Toxoplasmosis,Toxoplasmose cérébrale (Date),Cerebral Toxoplasmosis,Misc,Date,,
+e7800399-42d3-4dc6-af11-daa5f5682b2a,,,Cryptococcose neuro-méningée (Date),Cryptococcal neuro-meningeal disease (Date) ,Cryptococcose neuro-méningée (Date),Cryptococcal neuro-meningeal disease (Date) ,Misc,Date,,
+7b82d88a-f566-4934-b2ad-4f7c7bd48108,,,Méningite bactériennes (Date),Bacterial meningitis (Date) ,Méningite bactériennes (Date),Bacterial meningitis (Date) ,Misc,Date,,
+cb279713-3373-44be-91bf-ead37122d098,,,Autres Infections Oportunistes diagnostiquées,Other Oportunistic infections diagnosis,Autres Infections Oportunistes diagnostiquées,Other Oportunistic infections diagnosis,Misc,Text,,
+0d9692be-a4ea-48b8-bf44-3c6c8eb7ecf5,,,Date de diagnostic,Date of diagnosis ,Date de diagnostic,Date of diagnosis ,Misc,Date,,
+c7e66918-f4df-4f98-bd8e-b0ed5b12a4b2,dc10758d-06a2-43f1-b916-a111191e454a;e11eaac0-8c68-406e-aa78-34c2faf436d2;2a1c5417-6947-495b-a504-699eb08f224a;462e1a64-6d23-4590-9393-c5fdcba78a42;ecaf7d8e-161d-4e28-9ce8-b3b7dd9772bb;c6658a5c-91c9-4dd9-b9a2-d002a139dc97;20f7c749-9827-4508-a820-c134ebb3e271;175ae517-88f7-4c77-a07d-5bc149fcab3c;673ce6c2-9d44-48b2-8888-ef656cf1ccfd;e7800399-42d3-4dc6-af11-daa5f5682b2a;7b82d88a-f566-4934-b2ad-4f7c7bd48108;cb279713-3373-44be-91bf-ead37122d098;0d9692be-a4ea-48b8-bf44-3c6c8eb7ecf5,,VI. Infections Oportunistes diagnostiquées depuis la derniére consultation,VI. Oportunistic infections diagnosis date,VI. Infections Oportunistes diagnostiquées depuis la derniére consultation,VI. Oportunistic infections diagnosis date,Misc,N/A,,
+41cd339f-27fb-4acb-8841-74c9ea1069f1,74f83479-387d-46f9-8864-6835ea89c488;2f172c49-0200-4454-8be6-b80ec7e46b8a;5bddec2f-8561-4fe7-868b-801eee1ccb43;7d2ac701-7d53-4b66-a98a-1017b828ccb8;fa9d9a5c-82e1-4865-8376-b9bf4e403994;c7e66918-f4df-4f98-bd8e-b0ed5b12a4b2,,FICHE DU PATIENT AVEC VIH - ADULTE SUIVI,PATIENT WITH HIV - ADULT FOLLOWUP                ,FICHE DU PATIENT AVEC VIH - ADULTE SUIVI,PATIENT WITH HIV - ADULT FOLLOWUP                ,Misc,N/A,,
```

Comment:
@rubailly Please add this form to All Obs Templates

Reply:
Done.